src/hapi/pipelines/app/pipelines.py

```diff
@@ -75,5 +102,9 @@ def run(self):
     def output(self):
         self.locations.populate()
         self.admins.populate()
+        # TODO: Add hapi metadata from 3W (doesn't currently work as it's multiple datasets)
```

Comment:
metadata.populate() is expecting one dataset per scraper, so we could update metadata to be more flexible about the number of datasets/resources it can read per scraper.